db/scripts/media/createMediaTable.js

```diff
@@ -1,7 +1,7 @@
 import db from "../../connection.js";
 
 const mediaResponse = await db.query(
-   `CREATE TABLE IF NOT EXISTS media (id SERIAL PRIMARY KEY, aws_key TEXT, media_title TEXT, media_descr TEXT, date INT, location TEXT);`
+   `CREATE TABLE IF NOT EXISTS media (id SERIAL PRIMARY KEY, aws_key TEXT, media_title TEXT, media_descr TEXT, date INT, latitude DECIMAL, longitude DECIMAL);`
```

Comment:
should we save latitude and longitude in media table as well? Would we not save the in the locations table only? Should we record loc_id instead?
- also should be lat/lng instead of latitude / longitude